theseus_gui/src/pages/instance/Options.vue

```diff
@@ -298,55 +298,123 @@
     <div class="label">
       <h3>
-        <span class="label__title size-card-header">Instance management</span>
+        <span class="label__title size-card-header">Modpack</span>
       </h3>
     </div>
-    <div v-if="instance.metadata.linked_data" class="adjacent-input">
+    <div class="adjacent-input">
+      <label for="general-modpack-info">
+        <span class="label__description">
+          <strong>Modpack: </strong> {{ instance.metadata.name }}
+        </span>
+        <span class="label__description">
+          <strong>Version: </strong>
+          {{
+            installedVersionData.name.charAt(0).toUpperCase() + installedVersionData.name.slice(1)
+          }}
+        </span>
+      </label>
+    </div>
+    <div v-if="!instance.locked" class="adjacent-input">
+      <Card class="unlocked-instance">
+        This is an unlocked instance. There may be unexpected behaviour unintended by the modpack
+        creator.
+      </Card>
+    </div>
+    <div v-else class="adjacent-input">
+      <label for="repair-profile">
+        <span class="label__title">Unlock instance</span>
+        <span class="label__description">
+          Allows modifications to the instance, which allows you to add projects to the modpack. The
+          pack will remain linked, and you can still change versions. Only mods listed in the
+          modpack will be modified on version changes.
+        </span>
+      </label>
+      <Button id="repair-profile" @click="unlockProfile"> <XIcon /> Unlock </Button>
+    </div>
+
+    <div class="adjacent-input">
       <label for="repair-profile">
         <span class="label__title">Unpair instance</span>
         <span class="label__description">
-          Removes the link to an external modpack on the instance. This allows you to edit modpacks
-          you download through the browse page but you will not be able to update the instance from
-          a new version of a modpack if you do this.
+          Removes the link to an external Modrinth modpack on the instance. This allows you to edit
+          modpacks you download through the browse page but you will not be able to update the
+          instance from a new version of a modpack if you do this.
         </span>
       </label>
       <Button id="repair-profile" @click="unpairProfile"> <XIcon /> Unpair </Button>
     </div>
+
+    <div class="adjacent-input">
+      <label for="change-modpack-version">
+        <span class="label__title">Change modpack version</span>
+        <span class="label__description">
+          Changes to another version of the modpack, allowing upgrading or downgrading. This will
+          replace all files marked as relevant to the modpack.
+        </span>
+      </label>
+
+      <Button :disabled="inProgress || installing" @click="modpackVersionModal.show()">
+        <EditIcon />
+        Change modpack version
+      </Button>
+    </div>
     <div class="adjacent-input">
       <label for="repair-profile">
```

Comment:
Make sure the label ids and input ids match correctly. for these they are duplicated several times, which is bad for accessibility and causes clicking on the label to click on the wrong button